### PR TITLE
Update doc to reflect usage of tools project

### PR DESCRIPTION
**Pull request by trasher**
Minor other adaptations

http://glpi-plugins.readthedocs.io/en/feature-external-tools/empty/index.html

### CODE REVIEW

File: source/empty/index.rst [hunk 1]
```diff
@@ -62,10 +72,7 @@ Of course, the ``.travis.yml`` file can be pimped; you can run unit tests, creat
 Minifying CSS ans JS
 ^^^^^^^^^^^^^^^^^^^^
 
-A convenient script, using `Robo.li <http://robo.li>`_ is provided. You'll find two files related:
-
-* ``RoboFilePlugin.php`` in which are declared common usefull methods you may want to use (see below);
-* ``RoboFile.php`` an empty class, which extends ``RoboFilePlugin`` in which you can set you own stuff.
+A convenient script, using `Robo.li <http://robo.li>`_ is provided. The ``RoboFile.php`` file is an empty class that extends ``Glpi\Tools\RoboFile`` (provided by ``gpi-project/tools`` dependency) in which you can set you own stuff.
```
Comment (orthagh) [inline source/empty/index.rst]: gpi -> glpi
you own stuff -> your ...

File: source/empty/index.rst [hunk 2]
```diff
@@ -93,6 +100,8 @@ Just choose a target, and run something like:
 
    Also remember to adapt your scripts so they load your minified versions if available, and the original one otherwise :)
 
+   As of GLPI 9.2; you do not have to care about loading minified files when using ``add_css`` and ``add_javascript`` hooks!
```
Comment (orthagh) [inline source/empty/index.rst]: Maybe a precision of this point

File: source/empty/index.rst [hunk 3]
```diff
@@ -116,13 +125,17 @@ Once you get your updated ``PO`` files, you'll have to compile them to ``MO`` fi
...
 
-A release script is provided in ``tools/release``. This is a "simple" `Python <http://python.org>`_ script; you should just have Python installed on your system (this is instaleld by defautl on most linux distributions).
+A release script is provided in ``vendor/bin/plugin-release``. This is a "simple" `Python <http://python.org>`_ script; you should just have Python installed on your system (this is installed by default on most linux distributions).
 
-Using just the defaults, the script will try to retrieve the latest tag in your git repository, and will propose you to release it:
+.. warning::
+
+   Note that for the moment, the release script is only compatible if you use semantic versionning!
+
+Using just the defaults, the script will try to retrieve the latest tag in your git repository, add third party dependencies and create a `Release` on the github project:
```
Comment (orthagh) [inline source/empty/index.rst]: maybe in another part in this document, but for github, do we talk about needed api key ?

Comment (trasher) [inline source/empty/index.rst]: Yes, in the github release section:
http://glpi-plugins.readthedocs.io/en/feature-external-tools/empty/index.html#github-release

**Comment by trasher** [comment]
Fixed, thanks :)

**Pull request merged**